.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  test_e2e:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:10-browsers
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/mongo:3.4.4
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+
+      # Set up orient-express server.
+      - run: git clone https://github.com/lesfieres/orient-express.git
```

Comment:
I think we could use the last commit hash of the orient express to cache the whole folder:

For this, we would need a read-only account. User:Password could come from an ENV variable from the job on CircleCI.
`curl --silent  -u user:password https://api.github.com/repos/lesfieres/orient-express/commits | head -3 | tail -1 | cut -c 13-52 > orient-express-las-commit-hash`

then right before cloning the repo we could restore cache

`- restore_cache:
   keys:
        - v1-orient-express-{{ checksum "orient-express-las-commit-hash" }}`

then, after `cd..`, just save the cache for orient-express

`- save_cache:
          paths:
            - orient-express
          key: v1-orient-express-{{ checksum "orient-express-las-commit-hash" }}`